core/java/src/org/openda/blackbox/wrapper/BBStochModelInstance.java

```diff
@@ -1783,8 +1768,8 @@ private boolean isTimeIncludedInInterval(double currentTime, double targetTime,
 		return ((exchangeItemTime + 1.e-6) >= currentTime) && ((exchangeItemTime - 1.e-6) < targetTime);
 	}
 
-	private void addNoiseToExchangeItemForOneTimeStep(IPrevExchangeItem modelExchangeItem,
-													  IPrevExchangeItem noiseModelExchangeItem, int timeIndex, double[] noise, int transformationFromOperation, int stateSizeNoiseSizeRatio) {
+	private void addNoiseToExchangeItemForOneTimeStep(IExchangeItem modelExchangeItem,
```

Comment:
Codacy found an issue: [The method 'addNoiseToExchangeItemForOneTimeStep(IExchangeItem, IExchangeItem, int, double, int, int)' has an NPath complexity of 24990](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5102854)